### PR TITLE
Modal design

### PR DESCRIPTION
**Pull request by mandelatuks**
# The following were done:

- Add pop-up for the desktop version
- Add pop-up for mobile version
- Edit the work section in HTML so that can be rendered using  javascript
- Add style for both pop-ups.
- Add a JavaScript array to store all of the information for all projects.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -593,6 +798,262 @@ h3 {
...
+    top: 0;
+    bottom: 0;
+    overflow: auto !important;
+    background-color: #c1c7d0;
+    z-index: 9999;
+    width: 90%;
+    height: 80%;
+    padding: 100px 90px 60px;
+  }
+
```
Comment (MeqdamAlqudah) [inline styles.css]: - [ ] Please make sure that the popup window is responsive 

image

image


please fix this weird look of the close button it supoosed to look exactly as in the Figma design 

image

File: styles.css [hunk 2]
```diff
@@ -9,6 +9,204 @@ body {
...
+.closeimage {
+  display: flex;
+  align-self: flex-end;
+  justify-content: flex-end;
+  z-index: 999;
+  position: absolute;
+  right: 38px;
+  height: 20px;
+  cursor: pointer;
+}
```
Comment (MeqdamAlqudah) [inline styles.css]: - [ ] Please use the same close button as in Figma desing for mobile version notice that there is different close button for the mobile version and desktop version

| Your design | Figma design |
| ------------- | --------------  |
| image | image |
|

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @mandelatuks , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- no Linter errors 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mandelatuks** [comment]
@MeqdamAlqudah 
# changes made according to the review attached are the images of the changes
Screenshot 2022-04-08 221214
Screenshot 2022-04-08 221302
Screenshot 2022-04-08 221348

**Review by medaminedev66 — APPROVED** [review]
## Hi @mandelatuks, 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**